docs/2.5/admin-guide.md

```diff
@@ -1448,6 +1448,59 @@ To allow access for all users:
   * Update `sshd` configuration (usually `/etc/ssh/sshd_config`) to point to this
   file: `TrustedUserCAKeys /etc/ssh/teleport-user-ca.pub`
 
+## Certificate Rotation
+
+Take a look at the [Certificates chapter](architecture.md#certificates) in the
+architecture document to learn how the certificate rotation works. This section
+will show you how to implement certificate rotation in practice.
```

Comment:
it's not only certificate rotation, but it's certificate authority rotation - which means CA has been rotated as well, I think it's an important enough distinction to point it out here.

Reply:
yes, that's why I moved all of this into "how it works" section in the "architecture", where I (for the first time, actually) explicitly stated that there are TWO CAs and they're rotated.

Reply:
k, cool